test-it/ViaTests/JSReferenceCloseStatic.via

```diff
@@ -0,0 +1,32 @@
+define(StaticReferencesVI dv(VirtualInstrument (
+    Locals: c(
+        ce(dv(.ErrorCluster (true 100 'error' )) errorIn)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) err)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) getObjectReferenceError)
+
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) noErrorInAndValidReferenceError)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) errorInAndValidReferenceError)
+
+        e(.JavaScriptStaticRefNum staticReference)
+        e(dv(.Boolean false) isNotAValidRefnum1)
+        e(dv(.Boolean false) isNotAValidRefnum2)
+        e(.Occurrence occurrence)
+    )
+    clump (
+        // no error in, valid reference, close reference => no error out, reference not closed
+        JavaScriptInvoke(occurrence false getObjectReferenceError 'Date.now' staticReference)
+        CloseReference(staticReference noErrorInAndValidReferenceError)
```

Comment:
before closing assert the refnum is valid

Reply:
Done.